src/commands/setup.rs

```diff
@@ -80,13 +81,57 @@ impl Setup {
                     //Configure Bridge and veth_pairs
                     let status_block = network::core::Core::bridge_per_podman_network(
                         per_network_opts,
-                        &network,
+                        network,
                         &self.network_namespace_path,
                     )?;
-                    response.insert(net_name, status_block);
-                    // Setup basic firewall rules for each network.
-                    firewall_driver.setup_network(network)?;
-                    // TODO: Set up port forwarding. How? What network do we point to?
+                    response.insert(net_name.to_owned(), status_block);
+
+                    let id_network_hash = network::core_utils::CoreUtils::create_network_hash(
+                        net_name,
+                        MAX_HASH_SIZE,
+                    );
+
+                    // Need to enable sysctl localnet so that traffic can pass
+                    // through localhost to containers
+                    let network_interface = &network.network_interface;
+                    match network_interface {
+                        None => {}
+                        Some(i) => {
+                            let localnet_path = format!("net.ipv4.conf.{}.route_localnet", i);
+                            let sysctl_localnet = get_sysctl_value(localnet_path.as_str())?;
+                            if sysctl_localnet != *"1" {
+                                set_sysctl_value(localnet_path.as_str(), "1")?;
+                            }
+                        }
+                    }
+                    firewall_driver.setup_network(network.clone(), id_network_hash.clone())?;
+                    let port_bindings = network_options.port_mappings.clone();
```

Comment:
nit: Not a blocker: Would it be possible to move this inside `firewall` create as a new function and call and single function in setup.

Reply:
i think this is a good idea, just want to confirm with @mheon